<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GLSpectrumGUI</class>
 <widget class="QWidget" name="GLSpectrumGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>60</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Oscilloscope</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="controlBtns">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QDial" name="decay">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Decay:</string>
       </property>
       <property name="maximum">
        <number>240</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="holdoff">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Holdoff:</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="stroke">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Stroke:</string>
       </property>
       <property name="minimum">
        <number>4</number>
       </property>
       <property name="maximum">
        <number>240</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="histogram">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Display phosphor effect spectrum</string>
       </property>
       <property name="text">
        <string>Histogram</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/histogram.png</normaloff>:/histogram.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="maxHold">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Display max hold</string>
       </property>
       <property name="text">
        <string>Max Hold</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/maxhold.png</normaloff>:/maxhold.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="current">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Display live spectrum</string>
       </property>
       <property name="text">
        <string>Max Hold</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/current.png</normaloff>:/current.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="invert">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Exchange waterfall and histogram</string>
       </property>
       <property name="text">
        <string>Inv</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/invertspectrum.png</normaloff>:/invertspectrum.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="waterfall">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Display waterfall</string>
       </property>
       <property name="text">
        <string>Waterfall</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/waterfall.png</normaloff>:/waterfall.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="grid">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Toggle the scale grid</string>
       </property>
       <property name="text">
        <string>Grid</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/grid.png</normaloff>:/grid.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="gridIntensity">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Grid intensity</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="fftWindow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>FFT window function</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>Bartlett</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blackman-Harris</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Flat Top</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hamming</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hanning</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rectangle</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fftSize">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>FFT size</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>128</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>256</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>512</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4k</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="refLevel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Reference level (dB)</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="levelRange">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Range (dB)</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearSpectrum">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Clear spectrum histogram</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/res.qrc">
         <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="traceIntensity">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Trace intensity</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>histogram</tabstop>
  <tabstop>maxHold</tabstop>
  <tabstop>invert</tabstop>
  <tabstop>grid</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
